### DISCUSSION TITLE
actix-web 4.0.0 got panic "there is no timer running, must be called from the context of Tokio runtime"

### DISCUSSION THREAD

**Discussion by SardineFish** [actix/actix-web#1955] [Q&A] [answered]
I'm working on a project with `mongodb = 2.0.0-alpha` and `actix-web = 4.0.0-beta.1`, both depends on `tokio 1.x`. But got panic when using `mongodb` :
```
there is no timer running, must be called from the context of Tokio runtime
```
I tried to roll back these crates to some old version that depends on `tokio 0.2.x`, it works.
Is it a bug or something I was missing with the new `tokio` version?

**Comment by robjtede** [comment, marked as answer]
Beta 3 is out and might fix your issue.
